WordPress/WordPressTest/JetpackPluginOverlayViewModelTests.swift

```diff
@@ -1,6 +1,8 @@
 import XCTest
 @testable import WordPress
 
+
+
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/104b0c52a1168c8bc73da5fee16d942972ba7c48/WordPress/WordPressTest/JetpackPluginOverlayViewModelTests.swift#L5"></span>Limit vertical whitespace to a single empty line; currently 3<br />
<code>vertical_whitespace</code> <code>JetpackPluginOverlayViewModelTests.swift:5</code></td>
    </tr>
  </tbody>
</table>